src/waveform/renderers/allshader/dataaccess.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <QVector3D>
+#include <QVector>
+
+namespace allshader {
+template<typename T>
+class DataAccess {
+    T* m_pData{};
```

Comment:
Normally we put private members at the bottom of a class

Reply:
Sure, sure, still not used to that! I will change that.